doc/user_guide/getting_going.rst

```diff
@@ -68,7 +68,7 @@ on the PSyclone page or download and extract the latest release of
 PSyclone directly, e.g.
 
 .. parsed-literal::
-   > wget https://github.com/stfc/PSyclone/archive/\ |release|\ .tar.gz
+   > wget https://github.com/stfc/PSyclone/releases/\ |release|\ .tar.gz
```

Comment:
This will now be converted to a non-existing link: https://github.com/stfc/PSyclone/releases/2.1.0.tar.gz , could we revert to archive or was there any other reason for this change?

Reply:
I'm not sure why I did that - I think I decided that `archive` didn't exist but I've just tried and it's still there so I've undone this change. However, I also noticed that, for whatever reason, there are no releases listed under the 'releases' tab on GitHub so I've updated the doc to point to the `Tags` tab instead.

Reply:
Having put it back, I now find the `linkcheck` fails because the `release` tag doesn't get incorporated as part of the URL and then we get a 404 for https://github.com/stfc/PSyclone/archive/. I shall investigate.

Reply:
That wasn't fun - it seems that nesting markup isn't really supported in rst and no matter what I tried, I couldn't get the URL to generate correctly. I've therefore disabled the automatic generation of hyperlinks for the two cases by prefixing the URLs in question with a `\`. As they are no longer hyperlinks they are no longer invalid and don't bother the link checker.